requirements.txt

```diff
@@ -1,6 +1,6 @@
 boto3>=1.1.1
 csvkit>=0.9.1
-invoke>=0.10.1
+invoke==0.10.1
```

Comment:
As mentioned in #58, this is required to actually make any of the `invoke` commands (e.g. the server) work.